app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import (NotVaccinatedError,

```

Comment:
Code Style: It's better to use parentheses when importing multiple classes/modules. This makes your code more readable.